zerver/webhooks/newrelic/view.py

```diff
@@ -57,21 +57,28 @@ def api_newrelic_webhook(
     if info["owner"] != "":
         info["owner"] = "by **{}** ".format(info["owner"])
 
-    # These are the three promised current_state values
-    if "open" in info["status"]:
-        content = OPEN_TEMPLATE.format(**info)
-    elif "acknowledged" in info["status"]:
+    # These are the three promised state values
+    if "activated" in info["status"].lower():
```

Comment:
Why is these checking `in` rather than checking `info["status"].lower() == "activated"`? I see the old code did this sort of thing, but it seems very strange.

Reply:
@timabbott I kinda tried to maintain the old way here, I also find it strange why 'in' operator and not a string comparison. I will change that in a new commit.